src/features/utility-bar/ToggleViewportDropdown.tsx

```diff
@@ -1,40 +1,57 @@
+import { useEffect } from 'preact/hooks'
 import { AspectRatioIcon } from '@radix-ui/react-icons'
 import {
   screenSizes,
   useUtilityBarStore,
 } from '../../features/utility-bar/store'
 import { Dropdown } from '../../components/Dropdown'
 
-export default function () {
+interface ToggleViewportDropdownProps {
+  isDoc?: boolean
+}
+
+export default function (props: ToggleViewportDropdownProps) {
   const store = useUtilityBarStore()
 
+  // When on doc page, set active screen size to desktop
+  useEffect(() => {
+    if (props.isDoc) {
+      store.setActiveScreenSize(2)
```

Comment:
How about we use the Enum instead of the raw value.
```
store.setActiveScreenSize(ScreenSize.Desktop)
```

You'll have to import it from `utility-bar/store` up on line 5